Tests/FFI.lean

```diff
@@ -14,10 +16,13 @@ def nats : List Nat := [
   let u16 : UInt16 := .ofNat n
   let u32 : UInt32 := .ofNat n
   let u64 : UInt64 := .ofNat n
+  let x : ByteVector 2 := ⟨u16.toByteArrayC, sorry⟩
```

Comment:
These `sorry`s are kind of a red flag. @arthurpaulino Are you planning to remove them before merge?